runners/cli/src/main/kotlin/cli/main.kt

```diff
@@ -52,6 +52,11 @@ class GlobalArguments(args: Array<String>) : DokkaConfiguration {
         "Throw an exception if the generation exited with warnings"
     ).default(DokkaDefaults.failOnWarning)
 
+    override val globalMainPage by parser.option(
+        ArgType.String,
+        description = "Entry point for dokka multimodule documentation"
```

Comment:
IMHO this explains nothing, I'd write something like "Path to the global main page for multimodule documentation"